src/Select/SelectOption.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+
+class SelectOption extends React.Component {
```

Comment:
I think it's a bit strange that this component as merely a proxy for passing props which will be used in `Select`. In my opinion the `buildItemsArray` logic from `Select` should be here, since this is the "options" component.

Reply:
yeah, it currently is not much more, but it also is a first step in refactoring the dropdown and is resembles the "new" external API we have in mind for this refactoring. thats why wie convert the new children based interface to the old itemsArray one here.